Source/DafnyCore/DafnyPrelude.bpl

```diff
@@ -7,8 +7,9 @@
 // Copyright by the contributors to the Dafny Project
 // SPDX-License-Identifier: MIT
 
-const $$Language$Dafny: bool;  // To be recognizable to the ModelViewer as
-axiom $$Language$Dafny;        // coming from a Dafny program.
+const $$Language$Dafny: bool uses {  // To be recognizable to the ModelViewer as
+  axiom $$Language$Dafny;            // coming from a Dafny program.
```

Comment:
For your review: in addition to the described changes, I put this axiom inside a `uses` clause. Not sure of its implications.